galsim/chromatic.py

```diff
@@ -227,6 +460,152 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
         image += integral
         return image
 
+    def _interp_drawImage(self, bandpass, image=None, integrator=None, **kwargs):
+        """Draw method adapted to work for ChromaticImage instances for which interpolation between
+        stored images is being used.  Users should not call this routine directly, and should
+        instead interact with the `drawImage` method.
+        """
+        # Check whether the object has had some chromatic transformation applied to it.  Since we
+        # are interpolating between stored images, this is not a situation that _interp_drawImage
+        # can handle.  We have to lose the interpolation.
+        if hasattr(self, '_A') and self._chromaticTransformation(bandpass):
```

Comment:
Well, that's a good point.

I had been thinking that no chromatic transformations are okay, but technically we could accommodate a chromatic flux scaling by just incorporating it as pre-factors multiplying the stored images.  Do you think that makes sense?  If so, then some code refactoring will be necessary to allow this.